app/assets/v2/js/grants/_detail-component.js

```diff
@@ -615,7 +615,7 @@ Vue.component('grant-details', {
     vm.grant_salected_tags = vm.grant.grant_tags ? vm.grant.grant_tags.map(tag => tag.pk) : [];
     vm.chainId = vm.grant.tenants.length > 0 ? vm.grant.tenants[0].toLowerCase() : '';
 
-    const amount_received = Number(vm.grant.amount_received.replace(',', ''));
+    const amount_received = isNaN(vm.grant.amount_received) ? Number(vm.grant.amount_received.replace(',', '')) : vm.grant.amount_received;
```

Comment:
Just make sure that a `Number` instance is assigned:

```suggestion
    const amount_received = isNaN(vm.grant.amount_received) ? Number(vm.grant.amount_received.replace(',', '')) : Number(vm.grant.amount_received);
```